file_formats/ie_formats/ini.htm

```diff
@@ -332,60 +335,61 @@
+        <strong>spawn_facing_global = scope::variable_name</strong><br />
+        This attribute is only used if "point_select" is set to <span class="code">e</span> (POINT_SELECT_EXPLICIT). It expects a variable name that is used to retrieve the orientation for the spawned creature. <strong>Notes:</strong> Specifying the scope is mandatory for this attribute to work. Due to a bug in the implementation the orientation is read from "point_select_var" instead of this attribute. However, both attributes must be specified to work and require an explicit scope definition.<br />
         <br />
 
-        <strong>inc_spawn_point_index = ???</strong><br />
-        <span class="unknown">The meaning of this attribute is unknown.</span><br />
+        <strong>inc_spawn_point_index = boolean_value</strong><br />
+        This attribute is only used if "point_select" is set to <span class="code">i</span> (POINT_SELECT_INDEXED_SEQUENTIAL). It indicates that the variable defined by "point_select_var" is incremented on spawn location selection. If "create_qty" is greater than 1 then spawn point index is incremented after each creature is spawned. <strong>Note:</strong> The spawn operation applies a pre-selection and post-selection pass when executing a creature spawn section which results in the index being incremented twice if the same section is executed multiple times. Pre-selection is applied before the whole spawn operation is executed. Post-selection is applied after individual creature spawns which is important to note if "create_qty" is set to spawn multiple creatures at once.<br />
         <br />
 
-        <strong>hold_selected_point_key = ???</strong><br />
-        <span class="unknown">The meaning of this attribute is unknown.</span><br />
+        <strong>hold_selected_point_key = boolean_value</strong><br />
+        This attribute prevents the engine from updating the currently selected spawn location index directly after the spawn operation (post-selection). If "create_qty" is greater than 1 then this attribute forces the engine to reuse the initially selected spawn location for all spawns created during the current spawn operation. This attribute affects only sequential "point_select" modes which includes <span class="code">i</span>, <span class="code">r</span> and <span class="code">s</span>.<br />
         <br />
 
-        <strong>check_by_view_port = ???</strong><br />
-        The meaning of this attribute is not fully understood, but when on it limits spawning to outside of the current viewport (part of the area currently displayed). <i>ignore_can_see</i> also has to be set and at least in PSTEE, the spawnpoint has to be within visual range. It's unused in the originals under this name, but there are plenty of uses of <i>check_view_port</i> </span><br />
+        <strong>check_by_view_port = boolean_value</strong><br />
+        This attribute appears to be unimplemented.<br />
```

Comment:
This should still mention check_view_port, since there are so many uses.

Reply:
Done.